src/CreateAccount.js

```diff
@@ -11,9 +10,9 @@ class CreateAccount extends Component {
     const baseUrl = "/sign-up"
     const url = redirect ? `${baseUrl}?redirect=${redirect}` : baseUrl;
     return (
-      <Nav.Item className="px-3">
-        <Link to={url}>Create Account</Link>
-      </Nav.Item>
+      <Nav.Link href={url} >
```

Comment:
Why are we opting to do a full page refresh?

Reply:
no particular reason

Reply:
can update